### PR TITLE
Added Dejv (improved version)

### PR DESCRIPTION
**Pull request by CptToothpick**
Pridana funkcnost dejva
Zmena kodu v main pro nacitani cogs pres cyklus

### CODE REVIEW

File: cogs/dejv.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from disnake import ApplicationCommandInteraction, File, Message
+from disnake.ext import commands
+
+#global vars
+
+ACTIVATION_STRINGS = ["zavoláme dejva","zavolame dejva","zavoláme na to dejva","zavolame na to dejva","na to zavolame dejva","hej, dejve!","dejve, pojd to opravit"]
+DEJV_IMAGE_ROUTE = "./assets/dejvOpravar.png"
+DEJV_REPAIR_STRINGS = ["Přicházá Dejv a jde to opravit", "Zde jde Dave, naše spása", "Uhněte z cesty, přichází Dejv","Znáte Dejva, všechno spraví!"]
+
+#class definition
```
Comment (solumath) [inline cogs/dejv.py:10]: unnecessary comment

File: cogs/dejv.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#importing libs
+
+from random import randint, random
+from disnake import ApplicationCommandInteraction, File, Message
+from disnake.ext import commands
+
+#global vars
+
+ACTIVATION_STRINGS = ["zavoláme dejva","zavolame dejva","zavoláme na to dejva","zavolame na to dejva","na to zavolame dejva","hej, dejve!","dejve, pojd to opravit"]
```
Comment (solumath) [inline cogs/dejv.py:6]: could be in seperate file for messages and phrases.

File: cogs/dejv.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#importing libs
+
+from random import randint, random
+from disnake import ApplicationCommandInteraction, File, Message
+from disnake.ext import commands
+
+#global vars
```
Comment (solumath) [inline cogs/dejv.py]: unnecessary comment

File: cogs/dejv.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        if nalezeno:
+            with open(DEJV_IMAGE_ROUTE, "rb") as f:
+                await message.reply(DEJV_REPAIR_STRINGS[randint(0,len(DEJV_REPAIR_STRINGS)-1)],
+                    file = File(
+                        fp = f,
+                    )
+                )
+
+def setup(bot: commands.Bot):
+    bot.add_cog(Dejv(bot))
```
Comment (solumath) [inline cogs/dejv.py:36]: missing empty line

File: cogs/dejv.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
+#importing libs
```
Comment (solumath) [inline cogs/dejv.py]: unnecessary comment

**Review by lukynmatuska — APPROVED** [review]
Looks fine.

**Review by solumath — APPROVED** [review]

**Pull request merged**